docs/developer/tips-and-tricks.md

```diff
@@ -0,0 +1,41 @@
+# Tips and Tricks
+
+## Finding examples and documentation
+
+There are a number of examples you can try out on the https://arcade.makecode.com homepage.
+We also have several courses in development that are located [here](https://arcade.makecode.com/courses).
+API reference docs are located [here](https://arcade.makecode.com/reference).
+
+## "JavaScript" == "TypeScript"
+
+Despite the name shown in the MakeCode editor, our text language is actually a stricter subset of
+TypeScript. More information on what's supported can be found in the [language docs](https://makecode.com/language).
```

Comment:
Maybe say "a strict subset of TypeScript" to imply JS to static TS rather than TS to static TS.